pages/styles/login.module.scss

```diff
@@ -0,0 +1,42 @@
+@import "constant";
+
+.container {
+  @extend .grid--two-one;
+  position: relative;
+  height: 100vh;
+}
+
+.left {
+  background: linear-gradient(rgba(0, 0, 0, 0.6), rgba(0, 0, 0, 0.6)),
+    url("https://i.pinimg.com/originals/51/cd/d6/51cdd66f1768b8e380fd1eab073d532f.jpg");
+  background-size: cover;
+  height: 100%;
+}
+
+.right {
+  background: white;
+  position: relative;
+
+  .logo {
+    transform: translate(0%,55%);
+  }
+
+  & .form {
+    width: 80%;
+    height: 25%;
+    position: absolute;
```

Comment:
don't use position absolute for things that's not floating. 

Normally position absolute are just to force things to stay somewhere in the page (when it has to be on top of other contents).

> The trade-off (and most important thing to remember) about absolute positioning is that these elements are removed from the flow of elements on the page. An element with this type of positioning is not affected by other elements and it doesn’t affect other elements. This is a serious thing to consider every time you use absolute positioning. Its overuse or improper use can limit the flexibility of your site.

https://css-tricks.com/absolute-relative-fixed-positioining-how-do-they-differ/